__tests__/snapshots.test.js

```diff
@@ -16,16 +17,16 @@ for(const pkg of packageDirs) {
 	const storiesDir = path.resolve(pkgDir, 'stories');
 
 	if(fs.existsSync(storiesDir)) {
-		const { package: pkg, stories, component } = require(storiesDir);
-		const { presets = {default: {}} } = require(pkgDir);
+		const { package: pkg, stories, component: Component } = require(storiesDir);
```

Comment:
I'm guessing the casing is no longer important without the JSX syntax?

Reply:
Oh yes, of course.